airflow/dags/airtable_loader/california_transit_services.yml

```diff
@@ -9,6 +9,8 @@ id_name: service_id
 
 rename_fields:
   "# Static": num_static
+  # this typo exists in Airtable currently
+  "# Schedule FEeds": num_schedule_feeds
```

Comment:
I just fixed this in the Airtable to be "# Schedule Feeds", can just use the normal capitalization